daffodil-test-integration/src/test/scala/org/apache/daffodil/cliTest/TestCLIUdfs.scala

```diff
@@ -64,8 +64,9 @@ class TestCLIUdfs {
     val schema = path(
       "daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd"
     )
+    val classpath = udfClasspath()
 
-    runCLI(args"-v parse -s $schema -r user_func1") { cli =>
+    runCLI(args"-v parse -s $schema -r user_func1", classpath) { cli =>
```

Comment:
Was it intentional to not add the udfClasspath for this tests since this test is about not having any UDF's? Probably worth adding a comment to make that clear.

Though, I would have expected the CLI to succeed with UDF's on the classpath so maybe I'm missing something?

Reply:
So I think the addition of the daffodil-test as a dependency caused the internal classpath to have the UDFs available which caused the test to fail with the changes, so we had to explicitly set the the udfClasspath to be empty for it to pass.

Before we didn't need to because the daffodil-test (which contains the UDF dependencies was missing)

I can update with a comment for clarity though